wl/server/Command.java

```diff
@@ -20,318 +20,472 @@
- * The return value is a \n-terminated string.
+ * The return value is a <code>\n</code>-terminated string.
  *
+ * <p>
  * The following "protocol version" string has a very special meaning:
- * - munin
- *   The 'munin' protocol is used to print statistics about the server.
+ * <ul>
+ * <li> munin: <br>
+ *   The "munin" protocol is used to print statistics about the server.
  *   In the initial contact, language and username are skipped; instead the munin protocol
  *   version is printed (only currently supported version is 2).
  *   The password authentication is then performed like for registered users.
- *   If the password is correct, the server replies not ADMIN/SUCCESS but instead
- *   prints out server statistics in the following format:
- *   - Time in [1: milliseconds | 2+: hours] since the server was started, \n
- *   - {@literal Protocol version >= 2}: Average client lifetime in seconds, \n
- *   - Protocol version 1:
- *       - Number of current registered users, \n
- *       - Number of current unregistered users, \n
- *   - Protocol version 2:
- *       - Counter of registered users, \n
- *       - Counter of unregistered users, \n
- *   - Counter of unique registered users, \n
- *   - {@literal Protocol version <= 1}: Counter of successful connection attempts, \n
- *   - Counter of unsuccessful connection attempts, \n
- *   - Counter of CMD_LIST              requests, \n
- *   - Counter of CMD_INFO              requests, \n
- *   - Counter of CMD_DOWNLOAD          requests, \n
- *   - Counter of CMD_I18N              requests, \n
- *   - Counter of CMD_SCREENSHOT        requests, \n
- *   - Counter of CMD_VOTE              requests, \n
- *   - Counter of CMD_GET_VOTE          requests, \n
- *   - Counter of CMD_COMMENT           requests, \n
- *   - Counter of CMD_EDIT_COMMENT      requests, \n
- *   - Counter of CMD_SUBMIT            requests, \n
- *   - Counter of CMD_SUBMIT_SCREENSHOT requests, \n
- *   - Counter of CMD_CONTACT           requests, \n
- *   - Counter of CMD_SETUP_TX          requests, \n
- *   - {@literal Protocol version >= 2}: Counter of CMD_ADMIN_DELETE    requests, \n
- *   - {@literal Protocol version >= 2}: Counter of CMD_ADMIN_VERIFY    requests, \n
- *   - {@literal Protocol version >= 2}: Counter of CMD_ADMIN_QUALITY   requests, \n
- *   - {@literal Protocol version >= 2}: Counter of CMD_ADMIN_SYNC_SAFE requests, \n
- *   - Counter of unsuccessful commands, \n
- *   - ENDOFSTREAM\n
+ *   If the password is correct, the server replies not <code>ADMIN</code>/<code>SUCCESS</code> but
+ * instead prints out server statistics in the following format: <ul> <li> Time in [1: milliseconds
+ * | 2+: hours] since the server was started, <code>\n</code> <li> {@literal Protocol version >= 2}:
+ * Average client lifetime in seconds, <code>\n</code> <li> Protocol version 1: <ul><li> Number of
+ * current registered users, <code>\n</code> <li> Number of current unregistered users,
```

Comment:
Hm, stupid clang-format...
Is there a way to disable formatting of doc comments?

Reply:
Ouch! 
There seems to be no standard way to skip javadoc comments so I disabled clang-format for the whole file. The rest of the documentation should be fine I hope…